packages/sdk/package.json

```diff
@@ -10,7 +10,7 @@
     "umd"
   ],
   "scripts": {
-    "build": "run-s create-validators build:*",
+    "build": "run-s build:*",
```

Comment:
also remove from package.json
1. "create-validator-ts" and "ajv" dependencies
2. and "create-validators" script

Reply:
done